### DISCUSSION TITLE
Is there any way to know how many [id].json files have been generated with Incremental Static Regeneration and fallback=true in production

### DISCUSSION THREAD

**Discussion by larbisahli** [vercel/next.js#18109] [Help] [answered]
I have an ecommerce website, and I am using Incremental Static Regeneration for updating products with a `fallback=true` in `getstaticpaths ` to add more [pg].json files ready for delivery (Pagination), but with time I want to find a way to know how many [pg].json files the server has generated so far.

- How to do it programmatically?

![Screenshot 2020-10-21 233559](https://user-images.githubusercontent.com/52937392/96794500-3d85e600-13f6-11eb-90c0-fa3112debe45.png)

**Comment by jamesmosier** [comment, marked as answer]
There is no built-in way to do this in Next. You would need to write a custom script to traverse the directory and parse the results.

**Reply by larbisahli** [reply]
How can I do that using node I tried to use globby and write an /api/ script that can parse the build folder and send me a json respond, but I am new to node I am a bit stuck.

Vercel build output seems different from running build on my local machine.
